halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/EchoService.java

```diff
@@ -95,5 +95,9 @@ public Type getType() {
     Map<String, String> env = new HashMap<>();
 
     public Settings() {}
+
+    public Settings(List<String> profiles) {
```

Comment:
Why not implement this in `SpringServiceSettings.java` instead of adding this constructor across the various services here?